src/components/progress-bar/__test__/index.test.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+/**
+ * Internal dependencies
+ */
+import ProgressBar from './../index';
+
+const outer = '.wc-free-shipping-progress-bar__outer';
+const inner = '.wc-free-shipping-progress-bar__inner';
+
+describe( 'The ProgressBar component', () => {
+	it( 'shows the classnames of the outer div correctly', () => {
+		render( <ProgressBar /> );
+		expect( document.querySelector( outer ) ).toBeInTheDocument();
+	} );
+
+	it( 'shows the classnames of the inner div correctly', () => {
+		render( <ProgressBar /> );
+		expect( document.querySelector( inner ) ).toBeInTheDocument();
+	} );
```

Comment:
This is not a blocker, just a general question/caveat.

We're not testing implementation details here, unless `.wc-free-shipping-progress-bar__outer` is part of your public API. I suppose given this is div, there isn't much we can to select it by, which I'm okay with. However, in future projects, consider other ways to test this. If this test if purely visual, maybe an image regression test would work better?

Reply:
> We're not testing implementation details here, unless .wc-free-shipping-progress-bar__outer is part of your public API. I suppose given this is div, there isn't much we can to select it by, which I'm okay with. However, in future projects, consider other ways to test this. If this test if purely visual, maybe an image regression test would work better?

Both `.wc-free-shipping-progress-bar__outer` and `.wc-free-shipping-progress-bar__inner` are needed to display the progress bar. The outer div defines the absolute height of the progress bar, is needed to show the progress in percentage. If one of these two CSS classes isn't loading or gets renamed, the progress bar will no longer be working correctly, That's why I added checks for the CSS classes in this case.

Thanks for pointing out the option of replacing them with visual regression tests. I looked up visual regression tests briefly, and they seem to require Puppeteer. As I don't have e2e tests in place at the moment, I'd go ahead and merge this PR as it is. But I'll add a note to the e2e issue to potentially replace the CSS class checks of this PR with a visual regression test.